File: primeiro_semestre/appc/Exercícios Python/While/Ex5.py
n = int(input("Digite o numero a receber o fatorial: "))

"""if (n > 0):
    i=n
    while (i > 1):
        nf = (n*(i-1))
        i-=1
        n = nf
    print(f"O fatorial é: {nf}")
else:
    nf = 1
    print(f"O fatorial é: {nf}")"""
i=2
nf=1
while (i <=n):
    nf *=i
    i+=1
print(f"O fatorial é: {nf}")

File: segundo_semestre/c/exercicios_prova/ex5.cpp
/* Faça um programa que calcule o reajuste do salário de um funcionário.
Considere que o funcionário deverá receber um reajuste de 15%, caso
seu salário seja menor que R$500,00; se o salário for maior ou igual a
R$500,00, mas menor ou igual a R$1000,00, seu reajuste será de
10%; caso seja ainda maior que R$1000,00, o reajuste deverá ser de
5%*/

#include <iostream>

using namespace std;

int main (){
    float salario_ant, salario_atual;
    cout<<"Digite o valor do salario: "; cin>>salario_ant;
    
    if (salario_ant > 1000){
        salario_atual = (salario_ant * 1.05);
        cout<<"Novo salário: "<<salario_atual;
    }
    else {
        if (salario_ant >= 500){
            salario_atual = (salario_ant * 1.10);
            cout<<"Novo salário: "<<salario_atual;
        }
        else{
            salario_atual = (salario_ant * 1.15);
            cout<<"Novo salário: "<<salario_atual;
        }
    }


    return 0;
}